rules/windows/defense_evasion_elastic_defend_inactive_threshold.toml

```
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2023/09/28"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/09/28"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects an elevated frequency of Elastic Defend network connectivity errors originating from a single host in
+a brief time span. Such occurrences may arise due to a configuration adjustment or, conceivably, due to adversaries
+obstructing the agent's communication with the server in order to inhibit telemetry and alert transmission.
+"""
+from = "now-9m"
+index = ["logs-*"]
```

In the description we scope this to elastic defend

```suggestion
index = ["logs-elastic_agent.endpoint_security*"]
```


but actually ... I am not sure if `endpoint_security` is standard or based on how the integration is configured. We could figure that out and scope it down or expand this to _any_ integration under `logs-*` and then add the `data_stream.dataset` to the threshold fields. Your call